auth-service/.env

```diff
@@ -0,0 +1 @@
+bipin_epam=TEST_PASSWORD
```

Comment:
please do not push .env nor secrets to repo, never (you can create .env.example to have placeholders)